version.go

```diff
@@ -27,7 +27,7 @@ const (
 	// Note: please update release_notes.md when you change these values.
 	appMajor uint = 0
 	appMinor uint = 21
-	appPatch uint = 0
+	appPatch uint = 1
```

Comment:
nit: no need to bump the version here, as we cut releases when we tag the `master` branch.

Reply:
Commit still needs to be removed